microcosm_elasticsearch/models.py

```diff
@@ -30,3 +69,7 @@ def __ne__(self, other):
 
     def __hash__(self):
         return id(self) if self.id is None else hash(self.id)
+
+
+class Model(ModelBase, metaclass=ModelMeta):
```

Comment:
Because of the metaclass logic (see top of file), I can't define the `__doctype_field__` attribute in the class that has `metaclass=ModelMeta`. I chose to define a `ModelBase` class that has all the logic, with its subclass `Model` having the right metaclass.

Reply:
@jessemyers no reason - while answering I realized my rationale didn't hold up, I changed it back.